src/main/java/mezz/jei/config/ClientConfig.java

```diff
@@ -202,4 +220,20 @@ private void syncSearchColorsConfig() {
 		final ColorNamer colorNamer = new ColorNamer(searchColorsMapBuilder.build());
 		Internal.setColorNamer(colorNamer);
 	}
+
+	private static Predicate<Object> validEnumElement(Class<? extends Enum<?>> enumClazz) {
```

Comment:
I'd prefer to call it `enumClass`, the zz thing gives me bad flashbacks to java textbooks that try giving too-short names to everything.